site/app/controllers/admin/AdminGradeableController.php

```diff
@@ -1875,4 +1877,74 @@ public function maxPoints($gradeable_id) {
         }
         $this->core->getOutput()->renderJsonError("Unknown gradeable");
     }
+
+    /**
+     * Loads config info for a gradeable to allow editing
+     * @Route("/courses/{_semester}/{_course}/gradeable/edit/load", methods={"POST"})
```

Comment:
change to PHP Route 8 annotations

Reply:
[23a1867](https://github.com/Submitty/Submitty/pull/10325/commits/23a1867a8acc2ed4fcafc810e0998c57d96a4eb3)